### ISSUE TITLE
Clickables: Links that open in new tabs/windows should be tagged for assistive technology

### ISSUE THREAD

**Issue by mpitblado** [gbif/hp-beaty-biodiversity-museum#76]
**Requirement:**
Links that open in a new tab or a new window should either have an "aria-label" attribute or a screen-reader only element explaining to screen-readers that this opens in a new tab.

**8 Failed Elements**

- [x] 1.
```html
<a href="/specimen/search?country=CA" target="_blank" class="feature-overlay" ></a>
```

- [x] 2.
```html
<a href="/specimen/search?country=CA" target="_blank" ><p><span data-ajax-url="https://api.gbif.org/v1/occurrence/search?institutionCode=BBM&amp;country=CA" class="ajax-is-loaded">420,703</span></p> </a>
```

- [x] 3.
```html
<a href="/specimen/search?view=MAP" target="_blank" class="feature-overlay" ></a>
```

- [x] 4.
```html
<a href="/specimen/search?view=MAP" target="_blank" ><p><span data-ajax-url="https://api.gbif.org/v1/occurrence/search?institutionCode=BBM&amp;hasCoordinate=true" class="ajax-is-loaded">454,892</span></p> </a>
```

- [x] 5.
```html
<a href="/about" target="_blank" class="feature-overlay" ></a>
```

- [x] 6.
```html
<a href="/about" target="_blank" ><p>1228</p> </a>
```

- [x] 7.
```html
<a href="/specimen/search?mediaType=StillImage&amp;view=GALLERY" target="_blank" class="feature-overlay" ></a>
```

- [x] 8.
```html
<a href="/specimen/search?mediaType=StillImage&amp;view=GALLERY" target="_blank" ><p><span data-ajax-url="https://api.gbif.org/v1/occurrence/search?institutionCode=BBM&amp;mediaType=stillImage" class="ajax-is-loaded">37,141</span></p> </a>
```

**Comment by mpitblado** [comment]
This appears to have also been fixed via #78!
